index.bs

```diff
@@ -2430,45 +2466,90 @@ To <dfn>check if an [=attribution source=] exceeds the time-based destination li
 [=attribution source=] |source|, run the following steps:
 
-1. Let |unexpiredSources| be all [=attribution rate-limit records=] |record| in the [=attribution rate-limit cache=] where all of the following are true:
+1. Let |matchingSources| be all [=attribution rate-limit records=] |record| in the [=attribution rate-limit cache=] where all of the following are true:
+
      * |record|'s [=attribution rate-limit record/scope=] is "<code>[=rate-limit scope/source=]</code>"
      * |record|'s [=attribution rate-limit record/source site=] and |source|'s [=attribution source/source site=] are equal
      * |record|'s [=attribution rate-limit record/reporting origin=] and |source|'s [=attribution source/reporting origin=] are [=same site=]
      * |record|'s [=attribution rate-limit record/expiry time=] is greater than |source|'s [=attribution source/source time=]
-1. Let |unexpiredDestinations| be a new [=set=].
-1. For each [=attribution rate-limit record=] |unexpiredRecord| of |unexpiredSources|:
-    1. [=set/Append=] |unexpiredRecord|'s [=attribution rate-limit record/attribution destination=] to |unexpiredDestinations|.
-1. Let |newDestinations| be the result of taking the [=set/union=] of |unexpiredDestinations| and |source|'s [=attribution source/attribution destinations=].
-1. Return whether |newDestinations|'s [=set/size=] is greater than the user agent's [=max destinations covered by unexpired sources=].
+     * The [=duration from=] |record|'s [=attribution rate-limit record/time=] and |source|'s [=attribution source/source time=]
+        is less than [=destinations-per-reporting-site rate-limit window=]
+1. Let |destinations| be the [=set/union=] of all [=attribution rate-limit record/attribution destination=] in |matchingSources|,
+    [=set/union|unioned=] with |source|'s [=attribution source/attribution destinations=].
+1. Return whether |destinations|'s [=set/size=] is greater than [=max destinations per reporting site per rate-limit window=].
+
+To <dfn>delete sources for unexpired destination limit</dfn> given a [=set=] of
+[=attribution source/source identifiers=] |sourcesToDelete| and a [=moment=] |now|:
+
+1. If |sourcesToDelete| [=set/is empty=], return.
+1. [=set/iterate|For each=] [=attribution source=] |source| of the [=attribution source cache=]:
+    1. [=set/Remove=] |source| from the [=attribution source cache=] if |sourcesToDelete|
+        [=set/contains=] |source|'s [=attribution source/source identifier=].
+1. [=set/iterate|For each=] [=event-level report=] |report| of the [=event-level report cache=]:
+    1. If |report|'s [=event-level report/trigger time=] is greater than or equal to |now|,
+        [=set/remove=] |report| from the [=event-level report cache=].
+
+        Note: This would only delete [=obtain a fake report|fake reports=] that would never be
+        scheduled if they were true [=event-level reports=].
```

Comment:
What does `scheduled` mean here? Do we mean `sent` instead?

Reply:
`scheduled` means scheduled but not actually sent yet. We only delete fake reports that haven't been delivered yet.

Reply:
Maybe `never be stored` then?

Reply:
SG, updated, thanks.